### PR TITLE
feat : single product and wishlist

### PR DESCRIPTION
**Pull request by ragini07**
Added wishlist and single product page

https://user-images.githubusercontent.com/40401373/153728955-c1b6b8e8-29a4-4b90-b3a2-f1f59383ae4e.mp4

### CODE REVIEW

File: ProductListing/productList.css [hunk 1]
```diff
@@ -75,7 +75,6 @@
     max-width: 14rem;
```
Comment (znjs) [inline ProductListing/productList.css:75]: if possible try using relative units

File: Wishlist/wishlist.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+.icon-black{
+    color: var(--font-primary-color);
+}
+.icon-black:hover{
+    font-size: large;
+}
+.wishlist-container{
+   margin: auto;
+   max-width: 50rem;
```
Comment (znjs) [inline Wishlist/wishlist.css:9]: if possible try using relative units

File: Wishlist/wishlist.html [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+                    <span class="card-badge-left">Best Seller</span>
+                    <span class="card-badge-right"><i class="fas fa-trash-alt icon-black"></i></span>
+                    <div class="text-container">
+                        <div class="title-wishlist">
+                            <div class="text-container-title">Purple</div>
+                        </div>
+                        <div class="text-container-desc">Plastic Outdoor Chair Pre-assembled</div>
+                        <div class="price">
+                            <div class="sale-price">₹749</div>
+                            <div class="mrp-price">₹1498</div>
```
Comment (znjs) [inline Wishlist/wishlist.html:114]: if possible use Unicode for special symbols ₹ => U+20B9 (&#8377;)

File: product/product.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+.divider{
+    background-color: var(--light-gray);
+    width: 100%;
+    border: none;
+    height: 0.1px;
+    margin: 0.5rem 0 1rem;
+}
+.extra-detail{
+    color: var(--font--secondary-color);
+    font-size: 12px;
```
Comment (znjs) [inline product/product.css:52]: if possible use relative units to make fonts responsive

**Review by znjs — APPROVED** [review]

**Review by narharikale — APPROVED** [review]
Seems good 👍

**Pull request merged**